Uplift/Views/Home/ActivitiesListCell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  ActivityListCell.swift
+//  Uplift
+//
+//  Created by Elvis Marcelo on 4/9/23.
+//  Copyright © 2023 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class ActivitiesListCell: ListCollectionViewCell<Activity, ActivityListItemCell> {
+
+    // MARK: - Public data vars
+    static let itemHeight: CGFloat = 110.0
```

Comment:
Make this private

Reply:
I need the height of the cell in order to set the height of the collectionView as it's horizontally scrollable and I don't want it taking up more space than the cell height, so I need it to be public